gatsby-config.js

```diff
@@ -23,6 +23,13 @@ module.exports = {
 							maxWidth: 690
 						}
 					},
+					{
+						resolve: 'gatsby-remark-embed-youtube',
+						options: {
+							width: 800,
+							height: 400
```

Comment:
Perhaps this is more of a question than a change request. Are both of these values mandatory? Is it possible to only supply one of them and have the other be "automatic"? We'll have better (more full) aspect ratios if this can auto-adjust given the dimensions of the video.

Reply:
To be honest, I don't know.  It was in the sample I copied.  It might not even be required?